config/peerpods/podvm/lib.sh

```diff
@@ -459,6 +463,82 @@ function validate_podvm_image() {
     echo "Checksum of the PodVM image: $(sha256sum "$image_path")"
 }
 
+# Function to convert local bootc container image to cloud image
+# Input:
+# 1. container_image_repo_url: The registry URL of the source container image.
+# 2. image_tag: The tag of the source container image.
+# 3. auth_json_file (can be empty): Path to the registry secret file to use for downloading the image.
+# 4. run_args: arguments for the podman run command that runs bootc image builder
+# 5. bib_args: arguments for bootc image builder itself
+# Output: based on input, either, qcow2 image at output/qcow2/disk.qcow2 or ami
+# see: https://github.com/osbuild/bootc-image-builder
+function bootc_image_builder_conversion() {
+    container_image_repo_url="${1}"
+    image_tag="${2}"
+    auth_json_file="${3}"
+    run_args="${4}"
+    bib_args="${5}"
+
+    # some VM customizations
+    if [[ -n "${BOOTC_BUILD_CONFIG}" ]]; then
+        echo "Using Custom Bootc Build Configuration"
+        echo "${BOOTC_BUILD_CONFIG}" >> ./custom-config.toml
```

Comment:
Here you want to copy the content of `${BOOTC_BUILD_CONFIG}`  to `./custom-config.toml`, so either `cp -f "${BOOTC_BUILD_CONFIG}" ./custom-config.toml` or  `cat "${BOOTC_BUILD_CONFIG}" > ./custom-config.toml`

Reply:
Ignore this comment pls. I set `BOOTC_BUILD_CONFIG` as the path to the config, but it should be the file's content....